src/controller/GameController.java

```diff
@@ -318,7 +330,7 @@ public static void main(String[] args) {
         Log.toFile("Competition phase = " + (data.competitionPhase == GameControlData.COMPETITION_PHASE_PLAYOFF ? "playoff" : "round robin"));
         Log.toFile("Competition type = "  + (data.competitionType == GameControlData.COMPETITION_TYPE_GENERAL_PENALTY_KICK ? "general penalty kick challenge" : "normal"));
         Log.toFile("Auto color change = " + data.colorChangeAuto);
-        Log.toFile("Using broadcast address " + (localAddress.getBroadcast() == null ? localAddress.getAddress() : localAddress.getBroadcast()));
+        Log.toFile("Using broadcast address " + broadcastAddress.toString());
```

Comment:
does it not automatically call `.toString()`?
```suggestion
        Log.toFile("Using broadcast address " + broadcastAddress);
```

Reply:
Ah, you are right. Force pushed